src/StringManipulation/NSSolutionTwo.java

```diff
@@ -0,0 +1,50 @@
+class NSSolutionTwo {
+    public boolean backspaceCompare(String S, String T) {
```

Comment:
This looks a good solution but kind of not happy with 3 loops. But it is still O(n) time and O(1) space. So good job 👍